README.md

```diff
@@ -33,6 +33,14 @@ app.get('/:id', rescue(async (req, res, next) => {
        .json(user)
 }))
 
+/**
+ * `rescue.all` insures thrown errors coming from any middleware inside the array will be passed to `next` callback.
+ */
+app.post('/login', rescue.all([
+  validateLogin,
+  loginController.login,
+]));
+
```

Comment:
I think we shouldn't mix the examples. Perhaps a dedicated section just for showing the use case of `rescue.all` would be better.

Also, it would be better if you include some code demonstrating the logic inside `validateLogin` and `login`, similar to the original example.